style.css

```diff
@@ -252,3 +216,138 @@ h2 {
   height: fit-content;
   margin-bottom: 10px;
 }
+
+.tag {
+  color: #6070ff;
+  border-radius: 12px;
+  background-color: #ebebff;
+  padding: 5px 10px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.spacer {
+  margin-top: 15px;
+}
+
+.about_section_lists {
+  display: block;
+}
+
+.section_header {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.dropdown {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-bottom: 20px;
+  margin-top: 20px;
+  justify-content: space-between;
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.dropdown span {
+  font-weight: bolder;
+  transform: rotate(90deg);
+}
+
+.dropdown:first-child {
+  border: none;
+}
+
+.dropdown:last-child {
+  margin-bottom: 40px;
+}
+
+.dropdown:first-child span {
+  transform: rotate(180deg);
+}
+
+.btn {
+  padding: 20px;
+  width: fit-content;
+  height: 24px;
+  font-weight: 500;
+  font-size: 17px;
+  color: #396df2;
+  font-style: normal;
+  border: 1px solid #396df2;
+  background-color: transparent;
+  border-radius: 8px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.03em;
+  transition: all 0.4s ease-in-out;
+}
+
+.btn:hover {
+  background-color: #6070ff;
+  color: white;
+}
+
+.btn:active {
+  background-color: #2230d2;
+  color: white;
+}
+
+.btn:disabled {
+  background-color: #fff;
+  color: #5e6c84;
+  border-color: #5e6c84;
+}
+
+.tags {
+  margin-bottom: 20px;
+}
+
+.first_project,
+.second_project,
+.third_project,
+.forth_project {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 60px;
+  border: 2px solid grey;
+  border-radius: 12px;
+  padding: 15px 15px;
+  background-color: #fff;
+}
```

Comment:
- [ ] Good job on creating a beautiful design :1st_place_medal: 
  however it would be better to match the Figma design mostly on the border color, :+1: 
 **Your design** 
![image](https://user-images.githubusercontent.com/102757126/198826243-928c6fb5-5315-421f-8f6e-6b254b802d2e.png)
**Figma design**
![image](https://user-images.githubusercontent.com/102757126/198826144-392faa49-5182-45f6-9330-4632f96a29aa.png)